build.gradle

```diff
@@ -167,7 +167,7 @@ signing {
 dependencies {
     api platform('com.exactpro.th2:bom:3.0.0')
     implementation 'com.exactpro.th2:grpc-check1:3.5.1'
```

Comment:
We need to add dependecy to a new grpc-common version in grpc-check1. And update the grpc-check1 version here